docs/fsharp/tutorials/type-providers/creating-a-type-provider.md

```diff
@@ -1085,52 +1048,41 @@ When you write your own type providers, you might want to use the following addi
 
-- `Exceptions and Diagnostics from Type Providers`
-<br />  All uses of all members from provided types may throw exceptions. In all cases, if a type provider throws an exception, the host compiler attributes the error to a specific type provider.
-<br />
-  - Type provider exceptions should never result in internal compiler errors.
-<br />
+### Caching Schema Information
 
-  - Type providers can't report warnings.
-<br />
+Providers must often cache access to schema information. The cached data should be stored by using a file name that's given as a static parameter or as user data. An example of schema caching is the `LocalSchemaFile` parameter in the type providers in the `FSharp.Data.TypeProviders` assembly. In the implementation of these providers, this static parameter directs the type provider to use the schema information in the specified local file instead of accessing the data source over the network. To use cached schema information, you must also set the static parameter `ForceUpdate` to `false`. You could use a similar technique to enable online and offline data access.
 
-  - When a type provider is hosted in the F# compiler, an F# development environment, or F# Interactive, all exceptions from that provider are caught. The Message property is always the error text, and no stack trace appears. If you’re going to throw an exception, you can throw the following examples:
-<br />
-    - `System.NotSupportedException`
-<br />
+### Backing Assembly
 
-    - `System.IO.IOException`
-<br />
+When you compile a .dll or .exe file, the backing .dll file for generated types is statically linked into the resulting assembly. This link is created by copying the Intermediate Language (IL) type definitions and any managed resources from the backing assembly into the final assembly. When you use F# Interactive, the backing .dll file isn't copied and is instead loaded directly into the F# Interactive process.
 
-    - `System.Exception`
-<br />
+### Exceptions and Diagnostics from Type Providers
 
+All uses of all members from provided types may throw exceptions. In all cases, if a type provider throws an exception, the host compiler attributes the error to a specific type provider.
+
+- Type provider exceptions should never result in internal compiler errors.
+
+- Type providers can't report warnings.
+
+- When a type provider is hosted in the F# compiler, an F# development environment, or F# Interactive, all exceptions from that provider are caught. The Message property is always the error text, and no stack trace appears. If you’re going to throw an exception, you can throw the following examples: `System.NotSupportedException`, `System.IO.IOException`,  `System.Exception`
```

Comment:
ideally the apis would use xref links to go to the API docs:
`<xref:System.NotSupportedException>`, etc. 

Remove extra space after the last comma